snapshot-tests/run-rules-snapshot.sh

```diff
@@ -1,57 +1,56 @@
 #!/bin/bash
 
-game=$1
-if [[ $game = "" || $game = "standard" ]]; then
+cleanup()
+{
+  game="$1"
+  rm -f "${game}"
+  rm -f "${game}.html"
+  rm -f "${game}_intro.html"
+  rm -f "${game}.html.strip"
+  rm -f rules-difference.txt
+  rm -f engine-output.txt
+}
+
+game="$1"
+if [[ "$game" = "" || "$game" = "standard" ]]; then
   game=standard
   executable=../standard/standard
 else
-  executable=../$game/$game
+  executable="../$game/$game"
 fi
 
-if [[ ! -f $executable ]]; then
+[ -e "${executable}" ] || executable="../build/$game"
```

Comment:
This should probably be -f not -e, since if there was somehow a directory in that location this would be all sort of broken.

Reply:
Yes, although -f doesn't handle symbolic links, it should be better. I'll change this.